.github/workflows/release.yml

```diff
@@ -0,0 +1,108 @@
+on:
+  push:
+    tags:
+    - 'v*'
+
+name: Upload Release Asset
+
+jobs:
+  build:
+    name: Upload Release Asset
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Build project
+        run: |
+          bash ./scripts/artifacts.sh
```

Comment:
```suggestion
        run: bash ./scripts/artifacts.sh
```

Reply:
To clarify:

```
run: bash ...
```

for a single line, 

```
run: |
  bash ...
  ...
```

for multiple lines.